test/unit/meta/has_abi.cpp

```diff
@@ -15,34 +15,38 @@
 
 TTS_CASE( "Check for detection of native ABI")
 {
-  TTS_EXPECT_NOT(( eve::has_native_abi_v<eve::wide<double, eve::fixed<32>>>              ));
-  TTS_EXPECT_NOT(( eve::has_native_abi_v<eve::logical<eve::wide<double, eve::fixed<32>>>>));
+  constexpr auto native = eve::wide<float>::static_size;
+
+  TTS_EXPECT_NOT(( eve::has_native_abi_v<eve::wide<float, eve::fixed<2*native>>>              ));
+  TTS_EXPECT_NOT(( eve::has_native_abi_v<eve::logical<eve::wide<float, eve::fixed<2*native>>>>));
 
 #if defined(EVE_NO_SIMD)
-  TTS_EXPECT_NOT( eve::has_native_abi_v<eve::wide<double>>                );
-  TTS_EXPECT_NOT( eve::has_native_abi_v<eve::logical<eve::wide<double>>>  );
+  TTS_EXPECT_NOT( eve::has_native_abi_v<eve::wide<float>>                );
+  TTS_EXPECT_NOT( eve::has_native_abi_v<eve::logical<eve::wide<float>>>  );
 #else
-  TTS_EXPECT( eve::has_native_abi_v<eve::wide<double>>                );
-  TTS_EXPECT( eve::has_native_abi_v<eve::logical<eve::wide<double>>>  );
+  TTS_EXPECT( eve::has_native_abi_v<eve::wide<float>>                );
+  TTS_EXPECT( eve::has_native_abi_v<eve::logical<eve::wide<float>>>  );
 #endif
 
-  TTS_EXPECT( eve::has_native_abi_v<double>                           );
-  TTS_EXPECT( eve::has_native_abi_v<eve::logical<double>>             );
+  TTS_EXPECT( eve::has_native_abi_v<float>                           );
+  TTS_EXPECT( eve::has_native_abi_v<eve::logical<float>>             );
```

Comment:
Eeh? This seems like a duplicate

Reply:
We check that scalar have indeed a native ABI.